vault/counters_test.go

```diff
@@ -23,14 +24,22 @@ func testCountActiveTokens(t *testing.T, c *Core, root string) int {
 	t.Helper()
 
 	rootCtx := namespace.RootContext(nil)
-	resp, err := c.HandleRequest(rootCtx, &logical.Request{
+	req := &logical.Request{
```

Comment:
Potentially newbie question. what is the advantage of this syntax over the old? Potentially just more explicit?

Reply:
needed to use the request var further down to verify the response

Reply:
ahhh gotcha.

Reply:
no problem, extremely valid reason